install/install.ps1

```diff
@@ -69,18 +76,22 @@ if (!$Version) {
     if (!$windowsAsset) {
         throw "Cannot find the windows Dapr CLI binary"
     }
-    $zipFileUrl = $windowsAsset.url
+    $zipFileUrl = $windowsAsset.browser_download_url
     $assetName = $windowsAsset.name
 } else {
     $assetName = "dapr_windows_amd64.zip"
     $zipFileUrl = "https://github.com/${GitHubOrg}/${GitHubRepo}/releases/download/v${Version}/${assetName}"
 }
 
+if ($CustomAssetPrefix) {
+    $zipFileUrl = $zipFileUrl.Replace("https://github.com/dapr/cli/releases/download", $CustomAssetPrefix)
+}
+
 $zipFilePath = $DaprRoot + "\" + $assetName
 Write-Output "Downloading $zipFileUrl ..."
 
 $githubHeader.Accept = "application/octet-stream"
-Invoke-WebRequest -Headers $githubHeader -Uri $zipFileUrl -OutFile $zipFilePath
```

Comment:
What if URL is from GitHub, don't this fail due to missing header? Or, at least have higher risk of being throttled.

Reply:
https://github.com/dapr/cli/blob/master/install/install.sh#L109

In fact, I thought this would work because I saw that in the install.sh version, no specific header was passed when downloading the file.
Also, I have tested locally that it works without passing this header to download resources from github.
If you think there is a better way to verify this, feel free to ask.

Reply:
And the browser_download_url version is the one used in install.sh, which is the domain name starting with github.com, not the one starting with api.github.com. So I think it should work.

Reply:
It might work for some time. On the other hand, throttling will happen pretty quickly without GitHub Auth token. Any reason for this to be removed?

Reply:
I think it works in the same way as install.sh that request to github.com directly instead of api.github.com.
If there is throttling, it should happen to current version of install.sh now published.

Reply:
We can fix `install.sh` in another PR. The fact that this problem exists in `install.sh` does not mean it should exist here. We are seeing issues with github API throttles in other parts of the code and I don't see a reason to add more to the problem.

Please, put the header back.